tools/register_all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+python service_mongo.py add questionanswering QA localhost 8082 text\&text_image "$PWD"/../lucida/questionanswering/class.txt
+python service_mongo.py add imagematching IMM localhost 8083 image\&text_image "$PWD"/../lucida/imagematching/class.txt
+python service_mongo.py add calendar CA localhost 8084 text "$PWD"/../lucida/calendar/class.txt
+python service_mongo.py add imageclassification IMC localhost 8085 image\&text_image "$PWD"/../lucida/imageclassification/class.txt
+python service_mongo.py add facerecognition FACE localhost 8086 image\&text_image "$PWD"/../lucida/facerecognition/class.txt
+python service_mongo.py add digitrecognition DIG localhost 8087 image\&text_image "$PWD"/../lucida/digitrecognition/class.txt
+python service_mongo.py add weather WE localhost 8088 text "$PWD"/../lucida/weather/class.txt
+python service_mongo.py add musicservice MS localhost 8089 text "$PWD"/../lucida/musicservice/class.txt
+
```

Comment:
In the future though, this file will not exist, right? The future Lucida will have no microservices initially and only gets them as added, so there are no "current microservices inside lucida like QA". For now this is OK, but when it becomes fully independent and dynamic, this will not be acceptable as this implies some default microservices (Which there will be none).